datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -492,34 +492,62 @@ CREATE TABLE dcsa_im_v3_0.event_classifier (
     event_classifier_description varchar(250) NULL
 );
 
+DROP TABLE IF EXISTS dcsa_im_v3_0.equipment_event_type CASCADE;
+CREATE TABLE dcsa_im_v3_0.equipment_event_type (
+    equipment_event_type_code varchar(4) PRIMARY KEY,
+    equipment_event_type_name varchar(100) NOT NULL,
+    equipment_event_type_description text NOT NULL
```

Comment:
Shouldn't wee keep this to varchar(250) to be aligned with the rest

Reply:
The test data description was exceeding the limit , hence I changed it. should I shorten the test data instead ?

Reply:
Wow - I guess we need to change it to varchar(500) and let Mapje know also - so she can update the IM

Reply:
Update it to 300, so that its not pointed out later that we may have increased a bit much.